r-package/grf/R/get_scores.R

```diff
@@ -0,0 +1,234 @@
+#' Compute doubly robust (AIPW) scores for average treatment effect estimation
+#' using a causal forest. Under regularity conditions, the average of the DR.scores
+#' is an efficient estimate of the average treatment effect.
+#'
+#' @param forest A trained causal forest.
+#' @param subset Specifies subset of the training examples over which we
+#'               estimate the ATE. WARNING: For valid statistical performance,
+#'               the subset should be defined only using features Xi, not using
+#'               the treatment Wi or the outcome Yi.
+#' @param debiasing.weights A vector of length n (or the subset length) of debiasing weights.
+#'               If NULL (default) they are obtained via inverse-propensity weighting.
+#'               
+#' @references Farrell, Max H. "Robust inference on average treatment effects with
+#'             possibly more covariates than observations." Journal of Econometrics
+#'             189(1), 2015.
+#' @references Robins, James M., and Andrea Rotnitzky. "Semiparametric efficiency in
+#'             multivariate regression models with missing data." Journal of the
+#'             American Statistical Association 90(429), 1995.
+#'
+#' @export
```

Comment:
Is it necessary to export these or can they be internal? (`#' @keywords internal`)